### PR TITLE
Favicon and more

### LINKED ISSUE #146: Logo unification
**Please describe what has to be done**
There's 3 versions of the logo in the frontend right now, should only be one, and should probably be named Poll_Buddy_Logo.png or similar. This will necessitate checking for all usages of the old logo. May be helpful to do the header first before this as that'll cut out a lot of usages.

### LINKED ISSUE #154: "React App" in title on most/all pages
**Please describe what has to be done**
Should change to a format like "Login - Poll Buddy"

### PR DESCRIPTION
**Pull request by neha-deshpande001**
### Description of change(s), including why:

-Changed favicon to the Poll Buddy logo
-Removed all frontend warnings
-Changed the website's display name on the tab from "React App" to "Poll Buddy" and each individual page name to "PAGENAME - Poll Buddy"
-Replaced all non-cloud logos to our cloud logo (logo unification). Made image imports consistent throughout every page and component.
-Made pollCode page logo centered and formatted
-Made the registerWithSchool buttons centered
-Added links from the login page to forgotPassword and registerDefault
-Added a link from the homepage to the registerDefault page
-Added a logout button on the myclasses page so that we can still access the login page
-Added the header to the pollCode page

### Type of change(s)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Closing issues:

- Closes #146 #153 #154 #155 #161 

### Screenshots (if frontend change) of before and after:
Below: the name of the tab is Poll Buddy and the favicon is the cloud logo
new favicon

Below: updated to the cloud logo on the homepage
changed logo1

Below: updated to the cloud logo on the pollCode page
changed logo2

Below: the registerWithSchool page components centered
regwithschool centered

### CODE REVIEW

**Comment by hbh7** [comment]
I've tweaked a few things with the favicon (basically more sizes for various browsers/devices to be all happy based on favicon.io), and looked into how to do a per-page header. Looks like we'll want:
```
...
export default class whateverthispageis extends Component {

    componentDidMount(){
        document.title = "Home - " + document.title;
    }

    render() {
... 
```
... on each page, replacing PAGETITLE with whatever that page is. That should give us unique page titles rather than them all being just Poll Buddy.

**Comment by hbh7** [comment]
I also moved the logo around into an images folder. Looks like the way React wants you to do images is through importing and then referencing it I guess. 
Somewhere at the top: `import new_logo from '../../images/Poll_Buddy_Logo.png';`
Later in the page: `logo`

These should probably just be named logo though rather than new_logo, and similar with Poll_Buddy_Logo, should probably just be logo.png for simplicity. If you wanted to run through and swap those around, that'd be great :)

**Comment by neha-deshpande001** [comment]
> I also moved the logo around into an images folder. Looks like the way React wants you to do images is through importing and then referencing it I guess.

I did some research and discovered that React only supports imports from inside the src/ directory, not the public/ directory. This means that we can't do `import logo from '../../../public/logo.png';`. We can still use `logo`. To fix this, we can either move the images inside src/ or add a symlink using npm. Moving the images to src/ would totally oppose the nice URL images on the website, but I don't feel like I'm qualified to mess with npm, so if anyone wants to take on this challenge, go for it.

**Comment by hbh7** [comment]
> I did some research and discovered that React only supports imports from inside the src/ directory, not the public/ directory.

Ye that's why I put the images folder in src lol. 

Symlinking sounds like a terribly complicated idea so let's not do that. 

I'm just basing my thoughts off of https://create-react-app.dev/docs/adding-images-fonts-and-files/

I guess importing rather than image src bundles it in the code so it reduces server load and probably enables better optimization somehow, so that feels like the way to go. I think the benefit is more React magic than obvious is my theory. 

There isn't a warning/error thrown either way, just a preferred style I guess. 

On a side note, if we ever use React Native, that's how they want you to do it. I think we were thinking of doing something other than that for our eventual app, but idk, may be useful.

File: PollBuddy-Server/frontend/src/pages/privacy/privacy.js [hunk 1]
```diff
@@ -1,30 +1,46 @@
...
+            Here's some information about privacy policies. Pretty much, we need to talk about what info we will take,
+            how we will use it, how we will store/protect it, cookies, etc. At one point in the near future, we should gather together and write a privacy policy.
+            <br/><br/>
+            https://www.privacypolicytemplate.net/
+            <br/>
+            https://www.opentracker.net/article/how-write-website-privacy-policy
+            <br/><br/>
+            After developing our privacy policy, it would be cool to write it all in a google doc/markdown/whatever file
+            and then just display it on this page. I'm not sure how plausible that would be, but editing would be much easier.
+            Regular text on this .js file is very difficult to update and maintain.
```
Comment (hbh7) [inline PollBuddy-Server/frontend/src/pages/privacy/privacy.js:39]: Yes, that's probably a good idea. Importing data from a text doc should be possible lol

**Pull request merged**